docs/source/API/core/builtinreducers/MinMaxLoc.rst

```diff
@@ -0,0 +1,108 @@
+``MinMaxLoc``
+=============
+
+.. role::cpp(code)
+    :language: cpp
+
+.. role:: cppkokkos(code)
+    :language: cppkokkos
+
+Specific implementation of `ReducerConcept <ReducerConcept.html>`_ storing both the minimum and maximum values with corresponding indicies
+
+Header File: ``<Kokkos_Core.hpp>``
+
+Usage
+-----
+
+.. code-block:: cpp
+        
+    MinMaxLoc<T,I,S>::value_type result;
+    parallel_reduce(N,Functor,MinMaxLoc<T,I,S>(result));
+
+Synopsis 
+--------
+
+.. code-block:: cpp
+
+    template<class Scalar, class Space>
+    class MinMaxLoc{
+        public:
+            typedef MinMaxLoc reducer;
+            typedef MinMaxLocScalar<typename std::remove_cv<Scalar>::type,
+                                    typename std::remove_cv<Index>::type> value_type;
+            typedef Kokkos::View<value_type, Space> result_view_type;
+            
+            KOKKOS_INLINE_FUNCTION
+            void join(value_type& dest, const value_type& src) const;
+
+            KOKKOS_INLINE_FUNCTION
+            void init(value_type& val) const;
+
+            KOKKOS_INLINE_FUNCTION
+            value_type& reference() const;
+
+            KOKKOS_INLINE_FUNCTION
+            result_view_type view() const;
+
+            KOKKOS_INLINE_FUNCTION
+            MinMaxLoc(value_type& value_);
+
+            KOKKOS_INLINE_FUNCTION
+            MinMaxLoc(const result_view_type& value_);
+    };
+
+Public Class Members
+--------------------
+
+Typedefs
+~~~~~~~~
+
+* ``reducer``: The self type.
+* ``value_type``: The reduction scalar type (specialization of `MinMaxLocScalar <MinMaxLocScalar.html>`_)
+* ``result_view_type``: A ``Kokkos::View`` referencing the reduction result 
+
+Constructors
+~~~~~~~~~~~~
+
+.. cppkokkos:kokkosinlinefunction:: MinMaxLoc(value_type& value_);
+
+* Constructs a reducer which references a local variable as its result location.  
```

Comment:
the indentation is wrong here and everywhere below. The text for each cppkokkos item should be indented more, something like: 

```
.. cppkokkos:kokkosinlinefunction:: MinMaxLoc(value_type& value_);

   * Constructs a reducer which references a local variable as its result location.  
```

Reply:
Done, indeed it's better!